stardog/templates/statefulset.yaml

```diff
@@ -94,6 +94,13 @@ spec:
               path: /admin/healthcheck
               port: stardog
             initialDelaySeconds: 15
+          livenessProbe:
+            httpGet:
+              path: /admin/alive
+              port: stardog
+            initialDelaySeconds: 7200
+            periodSeconds: 60
+            failureThreshold: 5
```

Comment:
Let's add a timeout of 60s and increase `failureThreshold` to 60

Reply:
Done 👍